### DISCUSSION TITLE
ja/docs/getting-started/installing/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#350] [Page comments]
# ja/docs/getting-started/installing/

https://scratchaddons.com/ja/docs/getting-started/installing/

**Comment by tomocyan-github** [comment]
zip形式でダウンロードをしましたが、特に何の変化もありませんでした。どうすればいいですか？

**Comment by scratchsyoudokuekidesuwa** [comment]
zip形式でダウンロードして新しいフォルダーを作って解凍しましたが何も起きませんどうすればいいでしょうか

**Reply by WorldLanguages** [reply]
Did you try downloading from the Chrome Web Store?

**Reply by scratchsyoudokuekidesuwa** [reply]
絶対にchromeウェブストアからダウンロードしないといけないんですか?

**Reply by BroJac5246** [reply]
The Chrome Webstore is the best place to get the extension because it will automatically update. If that won't work, however, then follow [these steps](https://bashvlas.com/blog/install-chrome-extension-in-developer-mode/) with the .zip file you downloaded.
